src/config/mod.rs

```diff
@@ -166,6 +167,8 @@ pub struct ProtocolMetaConfig {
     pub enable_config_updates: bool,
     pub enable_deposited_txs: bool,
     pub enable_full_derivation: bool,
+    /// Specular: The L2-predeploy L1 oracle contract address
+    pub l1_oracle: Address,
```

Comment:
lets just hardcode like with `SystemAccounts`?

Reply:
will fix in next pr